assets/queries/cloudFormation/filter_and_alarm_exist_for_aws_management_console_authentication_failures/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Filter_And_Alarm_Exist_For_AWS_Management_Console_Authentication_Failures",
+  "queryName": "Filter And Alarm Not Exists For AWS Management Console Authentication Failures",
```

Comment:
Change **directory name** and **id** to: `authentication_failure_without_alarms`
Change **query name** to: `Authentication Failure Without Alarms`